ci_framework/roles/os_must_gather/tasks/main.yml

```diff
@@ -46,7 +46,8 @@
           oc adm must-gather --image {{ cifmw_os_must_gather_image }}
           --timeout {{ cifmw_os_must_gather_timeout }}
           --host-network={{ cifmw_os_must_gather_host_network }}
-          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs > {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
+          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs
+          -- SOS_SERVICES= gather > {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
```

Comment:
Thank you @fmount for improving the must_gather role. Can we add a Ansible var to pass the list of sos_services (default should be empty) so that end user can use it while consuming the role?

Reply:
makes sense in case we are in a troubleshooting session or we need to gather sos reports as well.
Added in my last PS.